utils/build/docker/nodejs/express4/auth.js

```diff
@@ -0,0 +1,88 @@
+const LocalStrategy = require('passport-local').Strategy
+const BasicStrategy = require('passport-http').BasicStrategy
```

Comment:
```suggestion
'use strict'

const { LocalStrategy } = require('passport-local')
const { BasicStrategy } = require('passport-http')
```

Reply:
why didn't you add the 'use scrict' ?

Reply:
Bc I copied your suggestion to test it before committing, it didn't work then made the proper changes: `const { Strategy: LocalStrategy } = require('passport-local')` and the `use strict` but I forgot to commit it.